R/cirls.fit.R

```diff
@@ -93,9 +93,8 @@ cirls.fit <- function (x, y, weights = rep.int(1, nobs), start = NULL,
   dev.resids <- family$dev.resids
   aic <- family$aic
   mu.eta <- family$mu.eta
-  unless.null <- function(x, if.null) if (is.null(x)) if.null else x
-  valideta <- unless.null(family$valideta, function(eta) TRUE)
-  validmu <- unless.null(family$validmu, function(mu) TRUE)
+  valideta <- family$valideta %||% function(eta) TRUE
```

Comment:
Use of the new %||% operator. It does exactly what unless.null() does. You can check that it now replaces these lines in glm.fit()